client-src/elements/chromedash-feature-page.ts

```diff
@@ -420,6 +481,50 @@ export class ChromedashFeaturePage extends LitElement {
         </div>
       `);
     }
+    if (this.isFeatureOutdated()) {
+      if (this.userCanEdit()) {
+        warnings.push(html`
+          <div class="warning layout horizontal center">
+            <span class="tooltip" id="outdated-icon" title="Feature outdated ">
+              <iron-icon icon="chromestatus:error" data-tooltip></iron-icon>
+            </span>
+            <span>
+              Your feature hasn't been verified as accurate since&nbsp;
+              <sl-relative-time
+                date=${this.feature.accurate_as_of}
+              ></sl-relative-time
+              >, but it is scheduled to ship&nbsp;
```

Comment:
NBSP will prevent line breaks, so I (and prettier IIRC) tend to use 
```suggestion
              >, but it is scheduled to ship${" "}
```
But it's not a big deal.

Reply:
Aha interesting. Done!